app/src/main/res/layout/drawer_list_item.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+    <data>
+        <import type="android.view.View"/>
+        <variable
+            name="menu"
+            type="org.fossasia.openevent.app.main.menu.OpevMenu"/>
```

Comment:
Are we using a new class just for menu? I think these complexities can be heavily reduced if we follow standard material guidelines for creating NavView